tests/conftest.py

```diff
@@ -9,3 +13,39 @@ def test_file(tmpdir: LocalPath) -> str:
     with open(filename, "w", encoding="utf-8") as file:
         file.write('"""A multi-line\ndocstring."""')
     return str(filename)
+
+
+def make_a_formatter() -> Formatter:
```

Comment:
I'd prefer to just import from `_test_utils`. I tend to forget which functions and fixtures I defined in `conftest` and I don't like having to refer back to `conftest` while browsing my tests.

Reply:
actually this is my bad, I did not intend to stage this file